.github/workflows/python-publish.yml

```
@@ -0,0 +1,33 @@
+# This yml file will trigger a Github Actions event that builds and upload the
+# Python package to PiPy. This makes use of Twine and is triggered when a push
+# to the master branch occures. For more information see:
+# https://help.github.com/en/actions/language-and-framework-guides/using-python-with-github-actions#publishing-to-package-registries
+
+name: Upload Python Package
+
+on:
+  push:
+    tags: 'v*.*.*'
```

The [github guide](https://docs.github.com/en/actions/guides/building-and-testing-python#publishing-to-package-registries) suggests a different way to handle releases:
```suggestion
  release:
    types: [created]
```
Should we use this instead?

---

Oh yes that looks cleaner, sometimes I avoid this as it would catch all releases including release candidates but I think this is great for this repo